conf/defaults.ini

```diff
@@ -1927,6 +1927,13 @@ enable =
 # feature1 = true
 # feature2 = false
 
+[feature_toggles.openfeature]
+# This is EXPERIMENTAL. Please, do not use this section
+provider = static
+
+[feature_toggles.openfeature.context]
+# This is EXPERIMENTAL. Please, do not use this section
+
```

Comment:
since it's empty, idk if it's useful to have it

Reply:
maybe just a few commented-out examples? 🤷‍♂️